lib/publish.js

```diff
@@ -74,7 +74,8 @@ module.exports = async (pluginConfig, context) => {
         return;
       }
 
-      const fileName = asset.name || basename(filePath);
+      let fileName = asset.name || basename(filePath);
+      fileName = template(fileName)(context);
```

Comment:
Why using a `let` here?
This should be `const fileName = template(asset.name || basename(filePath))(context)` instead